src/components/InputTodo.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
```

Comment:
Kindly do not disable linters as they help you in catching errors or bad practices which you may introduce in your code unintentionally.

- For removing prop-types error, refer to this article
https://www.freecodecamp.org/learn/front-end-development-libraries/react/use-proptypes-to-define-the-props-you-expect

- For removing button-has-type error, you may just give a type attribute with any legal property to the button tag.